test/web-platform-tests.js

```diff
@@ -81,271 +81,271 @@ function read(file) {
   return fs.readFileSync(Path.resolve(__dirname, '..', file), 'utf8');
 }
 
-var testharness = require(__dirname + '/web-platform-tests/resources/testharness.js');
+// var testharness = require(__dirname + '/web-platform-tests/resources/testharness.js');
```

Comment:
Please uncomment this file. While all the tests don't pass, we do have some that we need and rely on.

Reply:
I'm probably missing some insights here. I was getting an error because that particular file doesn't exisit on the repo. Hense all my comments. 

Also to me that variable wasn't even used, the tests using the `harness` function were comment at the end of the file !